shared/uavobjectdefinition/hwaq32.xml

```diff
@@ -123,7 +123,7 @@
 		<field name="USB_HIDPort" units="function" type="enum" elements="1" options="USBTelemetry,Disabled" defaultvalue="USBTelemetry">
 			<description/>
 		</field>
-		<field name="USB_VCPPort" units="function" type="enum" elements="1" options="USBTelemetry,ComBridge,DebugConsole,Disabled" defaultvalue="Disabled">
+		<field name="USB_VCPPort" units="function" type="enum" elements="1" parent="HwShared.USB_VCPPort" defaultvalue="Disabled">
```

Comment:
When the "parent" sets a defaultvalue, does the "child" still need to set a defaultvalue?
Or should the defaultvalue for these 4 boards not be "MSP"?